Doc/library/tkinter.font.rst

```diff
@@ -47,7 +47,16 @@ The different font weights and slants are:
 
    .. method:: actual(option=None, displayof=None)
 
-      Return the attributes of the font.
+      Returns information about the actual attributes that are obtained when
+      the font is used on *displayof*; the values obtained may differ from
+      those given (and retrieved by :meth:`Font.cget`) due to
+      platform-dependent limitations (such as the availability of font
+      families and pointsizes). If *displayof* is omitted, it defaults to the
+      main window.
+
+      If *option* is specified, the value of just that attribute is returned;
+      if it is omitted, the return value is a dictionary of all the attributes
+      and their values. See above for a list of the possible attributes. [1]_
```

Comment:
This says "above" but the reference is to a separate page. Is "above" referencing the constructor's docs? If so, let's add a link to it.

Reply:
The "above" is referring to "additional keyword options". The link is to give reasonable credit to the man page as the entirety of the `actual` docs is just a modified version of the content found there (though am open to a more clear way of showing this).